controllers/lvmcluster_controller.go

```diff
@@ -149,8 +149,6 @@ func (r *LVMClusterReconciler) reconcile(ctx context.Context, instance *lvmv1alp
 		}
 		if err != nil {
 			return ctrl.Result{Requeue: true, RequeueAfter: time.Minute * 1}, err
```

Comment:
- I'm sorry in failing to understand the scenario
- Would you pls expand a bit more?

t1: cr, finaliser, lv exist
t2: cr deleted, finaliser exist, lv failed to delete
t3: cr deleted, finaliser exist, lv deleted

this PR just removes finaliser after t3 in next reconcile?

Reply:
confused as well. 
So now if the `r.processDelete` succeeds, the code flow will continue even after the `!instance.DeletionTimestamp.IsZero()` condition on line 137 ?

Reply:
Quite right. I'm closing this PR.